index.php

```diff
@@ -0,0 +1,14 @@
+<?php
+require_once 'DBconnect/DBconnect.php';
+require_once 'DBgetAllData/DBgetAllData.php';
+
+$db = DBconnect();
+$allTableData =DBgetAllData($db);
+
+require_once 'DisplayLocations/DisplayLocations.php';
+
+Echo DisplayLocations($allTableData);
```

Comment:
Always lowercase e on `echo`

```suggestion
echo DisplayLocations($allTableData);
```

Reply:
replaced